Python/View_and_Combine_STL/README.md

```diff
@@ -0,0 +1,24 @@
+# View and Combine STL
+
+STL files are an important part of 3D asset creation and management. STL files provide us with basic maps of 3D objects created/edited in applications like Blender/Unreal/Unity
+
+STL are mainly used to create 3D maps of environments like cityscapes and heightmaps of terrain. 
+
+## Running the Script
+* Create a virtual env
```

Comment:
Replace `Create a virtual env` to `Initialize and start a Virtual Environment`. If possible you can add a good resource on the topic as well as a hyperlink

Reply:
Resolved